### PR TITLE
[CAZ-2024] Add search endpoint for single chargeable vehicle

### PR DESCRIPTION
**Pull request by cruddasj**
This PR introduces an endpoint that can be used to query the chargeability of a single VRN against a particular CAZ. This is to be used on the Payment Matrix page to power the Search function.

### CODE REVIEW

**Comment by cruddasj** [comment]
FYI - this is to be merged down after the updates from @kingarj go into develop.

**Comment by cruddasj** [comment]
Okay - with https://github.com/InformedSolutions/JAQU-CAZ-Payments-API/pull/175 now merged, this PR is good to go.

@kingarj - please can you review today so this can be merged down?

File: src/it/java/uk/gov/caz/psr/RetrieveSingleAccountVehicleAndChargeabilityIT.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    mockAccountServiceChargesSingleVrnCall(ACCOUNT_ID, "CAS300", 200);
+    mockVccsComplianceCall("CAS300", "vehicle-compliance-response-single-zone.json", 200);
+
+    givenSingleAccountVehicleChargeRetrieval()
+      .forAccountId(ACCOUNT_ID)
+      .forCleanAirZoneId(ZONE)
+      .forVrn("CAS300")
+      .whenRequestIsMadeToRetrieveASingleChargeableAccountVehicle()
+      .then()
+      .responseIsReturnedWithHttpOkStatusCode();
```
Comment (kingarj) [inline src/it/java/uk/gov/caz/psr/RetrieveSingleAccountVehicleAndChargeabilityIT.java]: Could you add a call here to the assertion method to check that the data provided in the response is as expected?

Comment (cruddasj) [reply src/it/java/uk/gov/caz/psr/RetrieveSingleAccountVehicleAndChargeabilityIT.java]: Good shout - commit inbound 👍

**Pull request merged**